schema.graphql

```
@@ -172,6 +172,26 @@ type Community {
   websiteLink: String
 }
 
+type CommunityPg {
+  createdAt: String
+	facebookURL: String
```

ESLint

[error] Replace `↹` with `··`

(prettier/prettier)